Project URLS breaking deployment

#32 is preventing deployment:

```
HTTPError: 400 Bad Request from https://upload.pypi.org/legacy/
Invalid value for project_urls. Error: Use valid URL.
Error: Process completed with exit code 1.
```

I think it might be the single quotes in one of the URLS... maybe use this instead:
```
https://github.com/{{cookiecutter.github_username}}/{{cookiecutter.plugin_name}}/blob/master/README.md
```

@DragaDoncila, can you try to test this out locally and deploy a package to https://test.pypi.org/ ?

--- a/{{cookiecutter.plugin_name}}/setup.cfg
+++ b/{{cookiecutter.plugin_name}}/setup.cfg
@@ -34,7 +34,7 @@ classifiers =
     {%- endif %}
 project_urls =
     Bug Tracker = https://github.com/{{cookiecutter.github_username}}/{{cookiecutter.plugin_name}}/issues
-    Documentation = 'https://github.com/{{cookiecutter.github_username}}/{{cookiecutter.plugin_name}}#README.md'
+    Documentation = https://github.com/{{cookiecutter.github_username}}/{{cookiecutter.plugin_name}}#README.md
     Source Code = https://github.com/{{cookiecutter.github_username}}/{{cookiecutter.plugin_name}}
     User Support = https://github.com/{{cookiecutter.github_username}}/{{cookiecutter.plugin_name}}/issues
 

